test/e2e/specs/all-features/navbar/base-config.spec.js

```diff
@@ -57,39 +57,41 @@ describe('Navbar ', function() {
     it('should render a markdown pattern using proper HTML', function () {
         // in ci we don't have the same globus groups so the "show" ACL hides the 3rd link ("Records")
         var idx = (!process.env.CI ? 3 : 2);
         // option #4 has only markdownName defined
-        element.all(by.css('#navbar-menu > li.dropdown')).get(idx).element(by.css("a")).getAttribute('innerHTML').then(function (aInnerHTML) {
+        let lastMenuItem = menu.all(by.css('.chaise-nav-item')).get(idx);
+        lastMenuItem.element(by.css('a')).getAttribute('innerHTML').then(function (aInnerHTML) {
             expect(aInnerHTML.indexOf("<strong>")).toBeGreaterThan(-1, "name was used instead of markdownName");
         });
     });
 
     if (!process.env.CI) {
         var menuDropdowns, disabledSubMenuOptions;
-        it('should have a disabled "Records" link.', function () {
-            menuDropdowns = element.all(by.css('#navbar-menu > li.dropdown'));
+        
+        it("should have 4 top level dropdown menus", function() {
+            expect(menu.all(by.css('.chaise-nav-item')).count()).toBe(4, "number of top level dropdown menus is wrong");
+        });
 
-            expect(menuDropdowns.count()).toBe(4, "number of top level dropdown menus is wrong");
+        it('should have a disabled "Records" link.', function () {
             // get index 2 (3rd item)
-            expect(menuDropdowns.get(2).element(by.css("a.disable-link")).getText()).toBe("Records", "text is incorrect, may include caret");
+            expect(menu.element(by.css("a.disable-link")).getText()).toBe("Records", "text is incorrect, may include caret");            
         });
 
         it('should have a header and a disabled "Edit Existing Record" submenu link (no children).', function () {
-            var editMenu = menuDropdowns.get(3);
-            // need to open menu so it renders and has a value
-            editMenu.click().then(function () {
-                subMenuHeader = editMenu.all(by.css("span.chaise-dropdown-header"));
-                expect(subMenuHeader.get(0).getText()).toBe("For Mutating Data", "Sub menu header is incorrect or not showing");
-
+            //menu should still be open from previous test case
+            // chaise-nav-item represents top menus in the navbar.
+            let editMenu = menu.all(by.css('.chaise-nav-item')).get(3);
+            editMenu.click().then(function() {
+                expect(editMenu.all(by.css('.chaise-dropdown-header')).get(0).getText()).toBe("For Mutating Data", "Sub menu header is incorrect or not showing");
                 return editMenu.all(by.css("a.disable-link"));
             }).then(function (options) {
                 disabledSubMenuOptions = options;
-
                 expect(options.length).toBe(4, "some options are not shown properly");
                 expect(disabledSubMenuOptions[0].getText()).toBe("Edit Existing Record", "the wrong link is disabled or none were selected");
             });
```

Comment:
Can you add the catch here too? In this case, `done.fail()` should NOT be included since we aren't using it in this test case to "control" the flow of the tests. 
```
.catch(function (err) {
  console.log(err);
});
```